components/prism-markdown.js

```diff
@@ -137,3 +155,68 @@ Prism.languages.markdown['italic'].inside['bold'] = Prism.languages.markdown['bo
 Prism.languages.markdown['italic'].inside['strike'] = Prism.languages.markdown['strike'];
 Prism.languages.markdown['strike'].inside['bold'] = Prism.languages.markdown['bold'];
 Prism.languages.markdown['strike'].inside['italic'] = Prism.languages.markdown['italic'];
+
+Prism.hooks.add('after-tokenize', function (env) {
+	if (env.language !== 'markdown')
+		return;
```

Comment:
I'm not sure what our style policy is on this, but I generally prefer `{}` around `if` blocks like this.

Reply:
Yeah, looks like core uses `{}` with `if` blocks. Otherwise, code looks good. Love this improvement.